src/api/resources/delegates.js

```diff
@@ -29,9 +29,20 @@ export default class DelegatesResource extends APIResource {
 		this.getStandby = apiMethod({
 			method: GET,
 			defaultData: {
-				orderBy: 'rate:asc',
+				sort: 'rate:asc',
 				offset: 101,
 			},
 		}).bind(this);
+
+		this.getForgers = apiMethod({
+			method: GET,
+			path: '/forgers',
+		}).bind(this);
+
+		this.getForgingStatus = apiMethod({
```

Comment:
Typo: `getForgingStats`

Reply:
Although since this is confusing, maybe we want to call it `getForgingStatistics`?

Reply:
I see I got confused but wouldn't it be better to match API definition? `getForgingStats`

Reply:
Probably it would be better to update the API definition.

Reply:
https://github.com/LiskHQ/lisk/issues/1713